pavement.py

```diff
@@ -1,57 +1,87 @@
 import paver
 from paver.easy import *
 import paver.setuputils
-
 paver.setuputils.install_distutils_tasks()
-import os, sys
 from socket import gethostname
-import pkg_resources
+import os, sys
+from os import environ
+
 from sphinxcontrib import paverutils
 
+# new 7/2019 changes
+import pkg_resources
+from runestone import get_master_url
+
+sys.path.append(os.getcwd())
+
+home_dir = os.getcwd()
 
-######## CHANGE THIS ##########
 project_name = "overview"
-###############################
 
-master_url = "https://runestone.academy"
+#master_url = None
+#if master_url is None:
+#    if 'RSHOST' in os.environ:
```

Comment:
We use `black` as the master formatter for all our .py files now.  I don't see any substantive changes, so please take this out of the PR.